plugins/soundsourcem4a/soundsourcem4a.cpp

```diff
@@ -40,6 +40,16 @@ const SINT kNumberOfPrefetchFrames = 2112;
 // The TrackId is a 1-based index of the tracks in an MP4 file
 const u_int32_t kMinTrackId = 1;
 
+// According to various references DecoderConfigDescriptor.bufferSizeDB
+// is a 24-bit unsigned integer value.
```

Comment:
OK, the yes data field is able to store 24 bit. That means up to 16 MB buffer for one SampleBlock. 
That sounds incredible high. The SampleBlock size from real files is always below 1 KB. 
I was hoping we where able to reduce the maximum buffer even more. 
How could one design a portable mp4 player if it need to provide up to 16 MB RAM. 
But why is the field 24 bit though ... :-/